backend/include/utils/RandomString.h

```diff
@@ -0,0 +1,51 @@
+#ifndef _RANDOM_STRING_H_
+#define _RANDOM_STRING_H_
+
+#include <ctime>
+#include <iostream>
+#include <unistd.h>
+#include <random>
+
+namespace Backend {
+    std::string randomString(const int len) 
+    {
+        static const char alphanum[] =
+                "0123456789"
+                "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+                "abcdefghijklmnopqrstuvwxyz";
+        std::string tmp_s;
+        tmp_s.reserve(len);
+
+        for (int i = 0; i < len; ++i) {
+            tmp_s += alphanum[rand() % (sizeof(alphanum) - 1)];
```

Comment:
Das ist eine potentielle Sicherheitslücke, da so ein randomString als salt verwendet wird und die rand() dafür nicht sicher genug ist.

https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/rand?view=msvc-170

Reply:
fixed in newest commit